commands/bash/container/delete-namespace/delete-namespace

```diff
@@ -2,7 +2,7 @@
 set -eu -o pipefail
 
 function delete_namespace {
-    helm delete --purge $(helm list | awk '{print $1}' | grep -E "\-$1\b")
+    helm delete -n $1 $(helm list --short -n $1)
```

Comment:
helm is namespace scoped now

Reply:
will we still be appending the namespaces to the helm releases moving forward?